src/app/[locale]/specs/permission/en.mdx

```diff
@@ -0,0 +1,347 @@
+
+When included in a permission set, the audience parameter may be inherited from the overall set.
+
+Parameters:
+
+- `lxm` (string array, required, positional): NSID of API endpoints. Wildcard (`*`) is allowed and gives access to all endpoints. Partial wildcards are not supported (eg, can not use `com.example.*`).
+- `aud` (string, semi-required): audience of API requests, as a DID service reference: DID followed by required service type fragment, eg `did:web:api.example.com#srvtype`). Supports wildcard (`*`). Required in granular string representation, and required in lexicon JSON unless `inheritAud` is true.
+- `inheritAud` (boolean, optional): if true, `aud` does not need to be defined.
+
+Examples:
+
+```
+# Submit moderation reports to any service
+rpc:app.example.moderation.createReport?aud=*
+
+{
+  "type": "permission",
+  "resource": "rpc",
+  "lxm": ["app.example.moderation.createReport"],
+  "aud": ["*"]
+}
+
+# Call any RPC method on a specific service
+rpc?lxm=*&aud=did:web:api.example.com%23svc_appview
+
+{
+  "type": "permission",
+  "resource": "rpc",
+  "lxm": ["*"],
+  "aud": ["did:web:api.example.com#svc_appview"]
+}
+```
+
+## `blob`
+
+Ability to upload media files (blobs) to PDS.
+
+Parameters:
+
+- `accept` (array of strings, required): MIME types or partial MIME type glob patterns (`*/*` or `text/*` for example). Same syntax as the `accept` field in the `blob` lexicon type.
+
+Examples:
+
+```
+# Upload any type of blob
+blob:*/*
+
+{
+  "type": "permission",
+  "resource": "blob",
+  "accept": ["*/*"]
+}
+
+# Upload video or html
+blob?accept=video/*&accept=text/html
+
+{
+  "type": "permission",
+  "resource": "blob",
+  "accept": ["video/*", "text/html"]
+}
+```
+
+## `account`
+
+Control of PDS account hosting details, such as (private) account email.
+
+Permissions of this type can not be included in permission sets, and must be requested directly by client apps.
+
+Parameters:
+
+- `attr` (string, required): a component of account configuration. Wildcard is not supported.
+- `action` (string, optional): degree of control. Currently supports `read` or `manage`. If not specified, default is `read`.
+
+Attributes:
+
+- `email`: account email address. The `read` action makes the email and verification status visible. The `manage` action includes `read`, and also allows changing the email address.
+- `repo`: ability to update entire public repository using a CAR file. The `manage` action allows importing entire CAR files, for example during account migration. The `read` action does nothing.
+- `status`: account status in network. The `manage` action allows deactivating and re-activating the account. The `read` action does nothing. (NOTE: this attribute is not part of PDS reference implementation as of November 2025)
```

Comment:
IIRC we mentioned this would be performed through the account management interface and never through 3rd party apps/clients.